fe/fe-core/src/main/java/com/starrocks/connector/elasticsearch/ElasticsearchConnectorFactory.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.connector.elasticsearch;
+
+import com.starrocks.common.DdlException;
+import com.starrocks.connector.Connector;
+import com.starrocks.connector.ConnectorContext;
+import com.starrocks.connector.ConnectorFactory;
+
+public class ElasticsearchConnectorFactory
+        implements ConnectorFactory {
+    @Override
+    public Connector createConnector(ConnectorContext context) {
+        try {
+            return new ElasticsearchConnector(context);
+        } catch (DdlException e) {
+            throw new RuntimeException(e);
```

Comment:
Print some error logs. As far as possible not to use DdlException in the catalog, if necessary, to convert StarRocksConnectorException.